### PR TITLE
[data] Folder refactoring

### PR DESCRIPTION
**Pull request by AndrewShkrob**
Moved tests into a separate folder - `test_data`
* gpx/
* kml/
* osm/
* world_feed_integration_tests_data/
* broken_bookmarks.kmb.test

I can't move `minsk-pass.mwm` to the test_data folder yet. It requires a lot of changes in the code.

Moved fonts into a separate folder:
* 00_NotoNaskhArabic-Regular.ttf
* 00_NotoSansThai-Regular.ttf
* 00_NotoSerifDevanagari-Regular.ttf
* 01_dejavusans.ttf
* 02_droidsans-fallback.ttf
* 03_jomolhari-id-a3d.ttf
* 04_padauk.ttf
* 05_khmeros.ttf
* 06_code2000.ttf
* 07_roboto_medium.ttf
* fonts_blacklist.txt
* fonts_whitelist.txt
* unicode_blocks.txt

Changes tested on Desktop (MacOS), iPhone and Android

### CODE REVIEW

File: qt/CMakeLists.txt [hunk 1]
```diff
@@ -127,25 +127,25 @@ copy_resources(
...
+  fonts/01_dejavusans.ttf
+  fonts/02_droidsans-fallback.ttf
+  fonts/03_jomolhari-id-a3d.ttf
+  fonts/04_padauk.ttf
+  fonts/05_khmeros.ttf
+  fonts/06_code2000.ttf
+  fonts/07_roboto_medium.ttf
+  fonts/blacklist.txt
+  fonts/unicode_blocks.txt
+  fonts/whitelist.txt
```
Comment (AndrewShkrob) [inline qt/CMakeLists.txt:148]: Change to just `fonts`

File: .gitignore [hunk 2]
```diff
@@ -84,7 +84,7 @@ data/[0-9][0-9][0-9][0-9][0-9][0-9]
 data/gps_track.dat
```
[the remark this answers is not in the document]
Comment (AndrewShkrob) [reply .gitignore:84]: mess in the data folder

**Comment by pastk** [comment]
A bit of organization in the `data/` folder is good!

What are the downsides? 
Some loss of convenience in git history?

**Comment by AndrewShkrob** [comment]
@biodranik @rtsisyk I've rebased this PR. Merge?

**Comment by biodranik** [comment]
There are risks of breaking something without significant benefits...

**Comment by AndrewShkrob** [comment]
> There are risks of breaking something without significant benefits...

I don't want to argue about the benefits of refactoring, but as for breaking something...

This PR has 2 commits:
1. Moving test data into a separate folder
2. Moving fonts data into a separate folder

What could be broken by the first commit? Only tests. But they are passing :)

What could be broken by the second commit? (Well, everything) Fonts initialization in the core. But how to check if it's not broken? It's easy, because if the app runs it means there were no fails during the init.
F.e. if the app can't see the new folder:
```
Error reading uniblock description:  FileAbsentException organicmaps/platform/platform.cpp:161, "File fonts/unicode_blocks.txt doesn't exist in the scope wrf 
```

If the app can't see one of the fonts that should be present in the folder
```
drape/glyph_manager.cpp:411 GlyphManager(): Error reading font file = fonts/00_NotoNaskhArabic-Regular.ttf ; Reason = FileAbsentException organicmaps/platform/platform.cpp:161, "File fonts/00_NotoNaskhArabic-Regular.ttf doesn't exist in the scope wrf 
```

So, nothing is broken. I've rechecked it on all platforms once more.
And here is the diff from logs for ios (if you'd like to check it yourself) https://www.diffchecker.com/fXoDl4Yi/

**Comment by vng** [comment]
@AndrewShkrob Please, keep only test data moving, I will merge immediately :)
Let's keep fonts as-is, or in another discussion.

**Comment by AndrewShkrob** [comment]
@vng done

**Review by vng — APPROVED** [review]

**Pull request merged**